COMPILING.md

```diff
@@ -87,9 +87,17 @@ git clone --recursive https://github.com/cuberite/cuberite.git
 
 ### Building ###
 
-Follow the instructions at [CMake on Unix-based platforms](#cmake-on-unix-based-platforms).
+Run the following commands to build Cuberite:
+
+```
+mkdir Release
+cd Release
+cmake -DCMAKE_BUILD_TYPE=RELEASE ..
+make -j`nproc`
```

Comment:
I think this should be tested first. I remember my laptop used to run out of ram and resort to swap with `-j4` when compiling some program (PhantomJS?), drastically increasing compilation time. with `-j3` all was good.